automation/jenkins_build.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# Set env var
+ACCESS_KEY=$1
```

Comment:
Can read those vars from the environment directly instead of passing them as parameters?